app/visit/visitService.js

```diff
@@ -60,6 +61,17 @@
       }
     }
 
+    function deleteVisit(visit) {
+      return $http.delete(Bahmni.Common.Constants.visitUrl + '/' + visit.uuid, {params: {purge: true}})
```

Comment:
Why purge?

Reply:
I guess its because you can only delete a visit when there are no encounters for it. I thought we would not want to keep visits with no encounters. @Benilde I think this is not yet specified.